### PR TITLE
PR: Change style of navbar to be similar to our main website

### LINKED ISSUE #144: Make the navbar look as it does in the current stable theme
This is how the navbar looks in the stable version of our docs:

Selección_028

And this is how it looks in the development one:

Selección_029

I think we need to make them match so the navbar looks the same as the one in our main website too.

Also, the `Home` link in the development navbar is not pointing to `www.spyder-ide.org`

### PR DESCRIPTION
**Pull request by ccordoba12**
This is the new style

Selección_047

(I incorrectly thought this could be changed in the docs website. I'll revert my changes there).

Fixes spyder-ide/spyder-docs#144

### CODE REVIEW

**Comment by ccordoba12** [comment]
@dalthviz, did I do the necessary changes to the right file?

**Comment by dalthviz** [comment]
I was adding the more Spyder styles related changes to the `spyder.scss` but probably should be ok :+1:

**Comment by ccordoba12** [comment]
> I was adding the more Spyder styles related changes to the spyder.scss

I think that's a good idea @dalthviz! Let me check if it works before doing any additional changes though.

**Comment by ccordoba12** [comment]
@dalthviz, I think this one is ready.

File: src/scss/theme.scss [hunk 1]
```diff
@@ -33,7 +33,6 @@
 
 .navbar-light {
   background: #fff !important;
-  box-shadow: 0 0.125rem 0.25rem 0 rgba(0, 0, 0, 0.11);
```
Comment (dalthviz) [inline src/scss/theme.scss]: @ccordoba12 this line needs to be removed?

Comment (ccordoba12) [reply src/scss/theme.scss]: I think so, but I'll double check.

Comment (dalthviz) [reply src/scss/theme.scss]: Maybe `!important` in the `spyder.scss` could be added in case of conflict between the rules

**Comment by juanis2112** [comment]
@ccordoba12 can you please rebase so we can merge this one? Is there anything else missing?

File: src/scss/theme.scss (hunk 1 above)
Comment (ccordoba12) [reply src/scss/theme.scss]: Ok, will do.

File: package.json [hunk 2]
```diff
@@ -8,7 +8,7 @@
     "build:production": "webpack --config webpack.prod.js"
   },
   "devDependencies": {
-    "copy-webpack-plugin": "^5.1.1",
+    "copy-webpack-plugin": "^6.0.3",
```
Comment (ccordoba12) [inline package.json:11]: This was necessary to fix an npm vulnerability message.

**Review by dalthviz — APPROVED** [review]
@ccordoba12 LGTM :+1:

**Comment by ccordoba12** [comment]
Thanks @dalthviz! I'm merging this one because it's needed by @CAM-Gerlach and @juanis2112 said she's ok with it.

**Comment by juanis2112** [comment]
Thanks @ccordoba12

**Pull request merged**